lang/python/python3/patches/027-fix-host-build-libressl.patch

```diff
@@ -0,0 +1,35 @@
+--- a/Modules/_ssl.c
++++ b/Modules/_ssl.c
```

Comment:
Commit and neither commit description says anything about this patch. Is it part of upstream repository?